packages/checkbox/package.json

```diff
@@ -20,17 +20,16 @@
     "access": "public"
   },
   "dependencies": {
-    "react-transition-group": "^4.4.1",
+    "@leafygreen-ui/a11y": "^1.3.4",
+    "@leafygreen-ui/emotion": "^4.0.3",
     "@leafygreen-ui/hooks": "^7.3.3",
     "@leafygreen-ui/lib": "^10.0.0",
+    "@leafygreen-ui/palette": "^3.4.4",
     "@leafygreen-ui/tokens": "^1.4.0",
     "@leafygreen-ui/typography": "^15.1.0",
-    "@leafygreen-ui/a11y": "^1.3.4",
-    "@leafygreen-ui/emotion": "^4.0.3"
-  },
-  "devDependencies": {
-    "@leafygreen-ui/palette": "^3.4.4"
+    "react-transition-group": "^4.4.1"
```

Comment:
nit should this go first? i think above we had a next import statement before lg ones

Reply:
I think they go alphabetically by default